bot.py

```
@@ -0,0 +1,248 @@
+    async def on_ready() -> None:
+        """Prints a message when the bot is connected to Discord"""
+        global global_state
+
+        if bot.user is None:
+            print("log the relevant error")
+            return
+
+        print(f"{bot.user.name} has connected to Discord!")
+        print(f"Bot ID: {bot.user.id}")
+
+        bot_id = bot.user.id
```

Flake8 (7.2.0)

[error] local variable 'bot_id' is assigned to but never used

(F841)